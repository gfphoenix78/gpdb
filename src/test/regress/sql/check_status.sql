select dbid, content, status, role, preferred_role, mode from gp_segment_configuration where status = 'd' or preferred_role != role;
