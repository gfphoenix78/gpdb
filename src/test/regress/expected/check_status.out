select dbid, content, status, role, preferred_role, mode from gp_segment_configuration where status = 'd' or preferred_role != role;
 dbid | content | status | role | preferred_role | mode 
------+---------+--------+------+----------------+------
(0 rows)

